source/first-steps/interfaces.rst

```diff
@@ -0,0 +1,76 @@
+User interfaces
+==========================
+
+.. note::
+
+   The choice of the default interface as well as the visiblity of different modules can be configured in the profiles or from :doc:`the user's preferences <preferences>`.
+
+.. _interface-standard:
+
+Standard Interface
+--------------------
+
+This is the main interface of the application. All the modules are available, but there are limits depending on the profile settings.
+
+.. image:: images/standard-interface.png
+   :alt: GLPI standard interface
+   :align: center
+   :scale: 50%
+
+
+The home page of the standard interface provides a summary view for quick access to active elements (tickets, notes, planning, contracts, etc.).
+
+It is divided into 5 views:
+
+* *Dashboard*
+
+   A customizable dashboard
+* *Personal view*
+
+   Shows the tickets opened, rejected, to be processed, to be approved (for which the logged-in user is the requestor) or those on which this user has an interaction as a validator or technician (awaiting validation, in progress, to be closed, in waiting). This view also offers him current issues, his schedule and available notes.
+* *Group view*
+
+   Offers the same information about tickets and issues as the personal view, but using the groups to which it belongs as a reference. Depending on their profile, they may or may not have access to the tickets for which their groups are requesting.
+
+* *Global view*
+
+   Offers statistics on tickets and issues according to their status and on contracts according to their expiration date. It also offers the view of the latest additions of objects in GLPI. Depending on the configuration used, new tickets can also be presented in this tab.
+
+* *RSS feed*
+
+   Proposes the content of defined RSS feeds [Gérer les flux RSS](06_Module_Outils/04_Flux_RSS.rst "Les flux RSS se gèrent depuis le menu Outils > Flux RSS")).
+
+.. note::
+
+   For Personal and Group views, only tables with information are displayed.
+
+.. note::
+
+   Two security messages may be displayed when you log in for the first time after completing the installation procedure.
+
+   The first asks you to change the passwords of the accounts created by default in GLPI, the second asks you to delete the directory ``glpi/install``.
```

Comment:
to note for future works (as the current pr is for translation), we may have additionnal messages here (migrations required  mainly)